elastic/integrations
# [Zeek] Add NTP and Signature data streams

## What does this PR do?

Adds the NTP and Signature data streams that weren't transfer from elastic/beats#23772 and elastic/beats#24224.  Also found that the `zeek.connection.id` field was being left as an empty object `{}` so I added a remove for that.

## Checklist


## Author's Checklist



## How to test this PR locally

```
cd integrations/packages/zeek
elastic-package check && elastic-package stack down && elastic-package build && elastic-package stack up -d -v -p default && eval "$(elastic-package stack shellinit)" && elastic-package test  -v --data-streams ntp,signature,connection
```

## Related issues

- closes #1461 

## Screenshots
